content/en-us/reference/engine/datatypes/CFrame.yaml

```diff
@@ -743,120 +743,108 @@ methods:
     deprecation_message: ''
   - name: CFrame:ToWorldSpace
     summary: |
-      Returns one or more `Datatype.CFrame` objects transformed from object to
-      world space.
+      Receives one or more `Datatype.CFrame` objects and returns them transformed from object to world space.
     description: |
-      Returns one or more `Datatype.CFrame` objects transformed from object to
-      world space. Equivalent to:
+      Receives one or more `Datatype.CFrame` objects and returns them transformed from object to world space. Equivalent to:
 
       `CFrame * cf`
     parameters:
-      - name: cf
+      - name: ...
         type: CFrame
```

Comment:
```suggestion
        type: Tuple
```

Reply:
I think what you mean here is that it can take a tuple of CFrames, yes? And that it also returns a tuple of CFrames?

Reply:
In this case, we wouldn't want to use `<Variant>` since neither the input nor output can be a tuple of mixed types (all must be CFrames and all coming back are CFrames).

Reply:
This is super cool syntax -- wasn't aware we supported it.

Reply:
Here's an example of where a tuple of numbers is cited:
https://create.roblox.com/docs/reference/engine/libraries/utf8#codepoint